style.css

```diff
@@ -0,0 +1,15 @@
+.test{
+    background: red;
+}
+
+.text-list{
+    font-size:14px;
+}
+.video-caption{
+    font-size:17px;
+}
+
+a, a:hover, a:focus, a:active {
+    text-decoration: none;
+    color: inherit;
+}
```

Comment:
You can try to Use grid layout for better output

Reply:
I suggest you use words that are conversational and not orders. 
The above is phrased like an order. instead phrase the comments politely, as suggestions instead of orders. For example:
* This is a polite suggestion: *"I think it would be better to use semantic tags"*
* This reads as an order: *"Use semantic tags*

This is among your project requirements
<img width="1660" alt="Screenshot 2022-09-26 at 12 59 35" src="https://user-images.githubusercontent.com/48546992/192256798-9e89b804-bef7-4403-a392-ad90ebd5202d.png">

- [ ] Also this explanation is a bit ambiguous. I suggest phrasing it as a conversation

This is a clear comment:

 *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*